plugins/UsersManager/API.php

```diff
@@ -1612,7 +1612,7 @@ public function generateInviteLink($userLogin, $expiryInDays = 7, $passwordConfi
         Piwik::postEvent('UsersManager.inviteUser.generateInviteLinkToken', [$userLogin, $user['email']]);
 
         return SettingsPiwik::getPiwikUrl().'index.php?'.Url::getQueryStringFromParameters([
-                'module' => 'Login',
+                'module' => Piwik::getLoginPluginName(),
```

Comment:
This fixes the issue of wrong plugin being used while generating the copy invite user link